schedule/publiccloud/mau-extratests-publiccloud.yml

```diff
@@ -0,0 +1,70 @@
+---
+name: mau-extratests-publiccloud
+description: |
+  Run mau-extratests on public cloud instances
+schedule:
+  - boot/boot_to_desktop
+  - publiccloud/download_repos
+  - publiccloud/ssh_interactive_init
+  - publiccloud/register_system
+  - publiccloud/transfer_repos
+  - publiccloud/patch_and_reboot
+  - publiccloud/ssh_interactive_start
+  - publiccloud/instance_overview
+  - console/system_prepare
+  - console/prepare_test_data
+  - console/consoletest_setup
+  - console/check_os_release
+  - console/cleanup_qam_testrepos
+  - console/openvswitch
+  - console/shar
+  - console/sshd
+  - console/update_alternatives
+  - console/rpm
+  - console/slp
+  - console/pkcon
+  - console/libvorbis
+  - console/command_not_found
+  - console/openssl_alpn
+  - console/autoyast_removed
+  - console/cron
+  - console/syslog
+  - console/mta
+  - console/check_default_network_manager
+  - console/git
+  - console/cups
+  - console/java
+  - console/sqlite3
+  - console/gdb
+  - console/sysctl
+  - console/sysstat
+  - console/wget_ipv6
+  - console/ca_certificates_mozilla
+  - console/unzip
+  - console/gpg
+  - console/rsync
+  - console/clamav
+  - console/shells
+  - console/sudo
+  - console/dstat
+  - x11/evolution/evolution_prepare_servers
+  - console/mutt
```

Comment:
Let's remove this as @asmorodskyi suggested 😄

Reply:
Also not sure about that. @asmorodskyi wanted to remove some modules. But for me it was not clear which ones.

Reply:
see commit message for details